src/build.d

```diff
@@ -1480,6 +1486,75 @@ class Dependency
         }
     }
 
+     // Metadata required for parallel execution
+    private
+    {
+        shared uint builtDeps; /// amount of dependencies already built
```

Comment:
To be consistent with the naming, I think this is counting the number of deps where the `run` method has been called.  So maybe this should be called `depsRunCount`, with the coment `// number of dependencies that have been run`?

Reply:
It would be consistent with `run` but i think `builtDeps` is more self-explanatory.as one needs to know about`run` to understand `depsRunCount`